progress_stats/graphs.py

```diff
@@ -26,24 +27,18 @@
 
 _num_graphs = 0
 
-def report_alt(self, type=0):
+def report_alt(*args, **kwargs):
+    self = args[0]
+    old = kwargs['_old']
     # 0=days, 1=weeks, 2=months
-    self.type = type
-    txt = self.css % bg
-    txt += self._section(self.todayStats())
-    txt += self._section(self.dueGraph())
-    txt += self.repsGraphs()
-    txt += self._section(self.introductionGraph())
-    txt += self._section(self.ivlGraph())
-    txt += self._section(self.hourGraph())
-    txt += self._section(self.easeGraph())
-    txt += self._section(self.cardGraph())
+    txt = old(self)
+    txt = re.sub(self._section(self.footer()), '', txt)
```

Comment:
this section takes the txt variable from the 'old' data, and substitutes the output of the 'footer' section for an empty string; i.e. takes the 'footer' entry off of 'txt', to which we then add our progressGraph section, and add the footer back on.

More generalized than my first version, and leaves more room for other addons to mod CollectionStats.report if needed.